lib/setExpressConfigs.js

```diff
@@ -45,6 +46,14 @@ module.exports = function (app) {
   // set morgan
   app.set('morgan', morgan)
 
+  // enable express-session
+  app.use(session({
+    // warning: issues may occur if secret does not match what is used in `cookie-parser`
+    secret: 'secret key',
+    resave: false,
+    saveUninitialized: false
+  }))
```

Comment:
our config would need to be a default config with an option to override it if we do end up doing this